### PR TITLE
Docs cleanup

### PR DESCRIPTION
**Pull request by BenTheElder**
- v1alpha3 -> v1alpha4 in docs
- make more things copyable
- simplify config examples to focus on the relevant parts

### CODE REVIEW

**Comment by aojea** [comment]
How can I check the netlify preview?

**Comment by BenTheElder** [comment]
https://deploy-preview-1169--k8s-kind.netlify.com

**Comment by BenTheElder** [comment]
Sorry, went to lunch before posting that. Need to check the netlify settings for this soon

File: site/content/docs/user/known-issues.md [hunk 1]
```diff
@@ -40,26 +40,26 @@ This problem is related to a bug in [docker on macOS][for-mac#3663]
 
 If you see something like the following error message:
 ```bash
```
Comment (aojea) [inline site/content/docs/user/known-issues.md]: Why this is not.changed to codefromline too?

Comment (BenTheElder) [reply site/content/docs/user/known-issues.md]: Code from inline is copyable things to save or run.

This is sable output

Comment (BenTheElder) [reply site/content/docs/user/known-issues.md]: This is not something you should copy and paste

File: site/content/docs/user/quick-start.md [hunk 2]
```diff
@@ -367,14 +357,9 @@ Ensure you `systemctl restart docker` to pick up the changes.
 ```yaml
 # an ipv6 cluster
 kind: Cluster
-apiVersion: kind.sigs.k8s.io/v1alpha3
+apiVersion: kind.sigs.k8s.io/v1alpha4
```
Comment (amwat) [inline site/content/docs/user/quick-start.md]: kind.x-k8s.io/v1alpha4 here and everywhere else

Comment (BenTheElder) [reply site/content/docs/user/quick-start.md]: done

**Pull request merged**